onnxruntime/core/graph/graph_utils.h

```diff
@@ -187,5 +187,73 @@ void FinalizeNodeFusion(Graph& graph, Node& first_node, Node& second_node);
 */
 void FinalizeNodeFusion(Graph& graph, const std::vector<std::reference_wrapper<Node>>& nodes, Node& replacement_node);
 
+
+/*
+Find the first input edge of a node that the source node's operator type, version, and domain match the given values.
+@returns nullptr when not found.
+*/
+const Node::EdgeEnd*
+FindFirstInputEdge(
```

Comment:
Let's call this something like FindFirstMatchingInputEdge.
"Find the first input edge of a node that the source node's" -> "Find the first input edge of the given node such that the edge's source node has operator type, version, and domain that matches the given values."

Reply:
I'll remove this function for now.